### DISCUSSION TITLE
Need Help For 500 Vite Error in Nuxt3

### DISCUSSION THREAD

**Discussion by muhni97** [nuxt/nuxt#16441] [Questions] [answered]
Hello guys. 
When I try to install nuxt3, I am encountering a problem like this ;

This is from my console
GET http://localhost:3000/         HTTP/1.1 500 Vite Error 13ms]

This is from localhost:3000
No response returned from render handler: /__nuxt_error?url=/&statusCode=500&statusMessage=Vite+Error&message=[vite-node]+[ERR_INVALID_URL_SCHEME]+/node_modules/ufo/dist/index.mjs&stack=%3Cpre%3E%3Cspan+class=%22stack+internal%22%3Eat+/node_modules/ufo/dist/index.mjs%3C/span%3E%3C/pre%3E&data

My node version is 16.18.1 
Whenever I upgrade it to the latest version, the same problem still exist.
What should do for that?

**Comment by TheAlexLichter** [comment, marked as answer]
See https://github.com/nuxt/framework/issues/8706 - most likely remove special characters from your folder name / path for now.

**Reply by muhni97** [reply]
Thanks sir. I fixed the problem with [#8706](https://github.com/nuxt/framework/issues/8706)
